lib/std/zig/parser_test.zig

```diff
@@ -2837,6 +2837,36 @@ fn testParse(source: []const u8, allocator: *mem.Allocator, anything_changed: *b
     return buffer.toOwnedSlice();
 }
 
+fn printLine(line: []const u8) void {
+    warn("{}{}\n", .{ line, "\u{23CE}" }); // Carriage return symbol
```

Comment:
```suggestion
    warn("{}⏎\n", .{ line });
```
nit: we can put unicode in the source.

Reply:
Done in latest commit.

Reply:
Could you also change it in `printWithVisibleNewlines` for sake of consistency.